QuestionBot/QuestionBot/Model/ConsoleWrapper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace QuestionBot.Model {
+    public class ConsoleWrapper : IConsole {
```

Comment:
Very nice! This is a great way to make your module testable and will also allow us to easily change the input/output source in the future (to something like Slack for instance)